wordpress-mobile/AztecEditor-Android
# Issue/403 fixing list styling of empty line

Fixes #403 
Fixes #406


### Test 1

1. Load empty editor.
2. Press enter multiple times.
3. Put cursor somewhere in the middle, on an empty line.
4. Apply list style.
5. Switch to HTML mode and confirm that empty list with `<li>` tags in it is there.

### Test 2:

1. Load empty editor.
2. Press enter 3 times. Optionally put some text on any line except first.
3. Select all the text.
4. Apply quote.
5. Notice that quote is correctly applied to all selected lines.
